wp-graphql.php

```diff
@@ -684,6 +690,45 @@ public static function get_app_context() {
 
 			return $app_context;
 		}
+
+		/**
+		 * Adds default GraphQL post-types settings
+		 *
+		 * @param WP_Post_Type|array $args       Post type arguments.
+		 * @param string             $post_type  Post type
+		 * 
+		 * @return array
+		 */
+		public static function add_post_type_settings( $args, $post_type ) {
+			global $wp_post_types;
+
+			$post_type_settings = [
+				'show_in_graphql'                 => true,
+				'graphql_single_name'             => str_replace( '_', '', ucwords( strtolower( $post_type ), '_' ) ),
+				'graphql_plural_name'             => str_replace( '_', '', ucwords( strtolower( $post_type ), '_' ) ) . 's',
+				'register_schema_type'            => true,
```

Comment:
what's the use case for this so I can understand more?

Reply:
This would replace the primary use of the `skip-post-type-registry` WooGraphQL has been using to opt out of using the core PostObject types definition for WooCommerce post-type so it can roll it's own.

Reply:
See the example in the PR summary above to see how it will be use for `Products` in WooGraphQL. :point_up: